<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-G9M353E" failures="1" tests="1" name="runnerForFailedTestCases.FailedTestCases" time="4.270" errors="0" timestamp="2021-04-23T21:06:55 EET" skipped="0">
  <testcase classname="runnerForFailedTestCases.FailedTestCases" name="runScenario" time="4.270">
    <failure message="mohamed expected [true] but found [false]" type="java.lang.AssertionError">
        <![CDATA[java.lang.AssertionError: mohamed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at stepsForFailedTestCases.FailedSteps.user_enter_valid_and_for_names_fields(RegistrationTest.java:38)
at âœ½.User enter valid "mohamed" and "Ahmad" for Names Fields(file:///E:/Automation%20Task%20360Imaging/GUI_Testing_Project/src/test/java/features/FailedTestCasesForRegisterAndLogin.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runnerForFailedTestCases.FailedTestCases -->
